vnext/ReactUWP/Views/FlyoutViewManager.cpp

```diff
@@ -216,10 +237,17 @@ void FlyoutShadowNode::updateProperties(const folly::dynamic &&props) {
           popup.IsLightDismissEnabled(m_isLightDismissEnabled);
       }
     } else if (propertyName == "isOpen") {
-      if (propertyValue.isBool()) {
+      if (propertyValue.isBool() && !m_isContextFlyout) {
```

Comment:
This check doesn't seem right. There is a good chance that you're trying to set isOpen before m_isContextFlyout is set. So you want to remove/move this out of the for loop.